gitter/newcomer_messages.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+import requests
+
+
+def get_newcomer_messages():
+    logger = logging.getLogger(__name__)
+    IMPORT_URL = 'https://pastebin.com/raw/TR4YaaU8'
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
```

Comment:
Not quite understand why simply `return` here. Would the program do anything meaningful even if it doesn't get any message?